packages/react-components/src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -18,24 +18,23 @@ class OMBInfo extends React.Component {
     this.setState({ modalOpen: false });
   };
 
-  modalContents = minutes => (
+  modalContents = (minutes, resBurdenName = 'education benefits') => (
```

Comment:
I'm wondering if it would be better to have a more generic default, like "benefits". That would be a slight change to 27 `<OMBInfo>` components in `vets-website` once the component-library version got updated, but I don't think it would be a bad change.

Reply:
I think that's a great suggestion. Who can I reach out to that can make this type of decision or can we? A more generic default would provide a lot of value as teams wouldn't have to update their OMBInfo components unless they want to override it.

Reply:
I think our team can make that decision as long as we make sure that it's captured in the release notes.

Reply:
I agree, I think we can make this decision, but we need to reach out to the team responsible and make sure they are aware and work with them on updating their code. And in fact, it might be better if we just update their code and they review it. So once this gets merged, I'd say go ahead and do a release and then submit a PR in vets-website that bumps the component-library version and updates all uses of this component so that they still read "education benefits"